api/src/main/java/org/openmrs/api/db/CohortDAO.java

```diff
@@ -52,12 +53,14 @@
 	 * Finds all cohorts that contain the given patientId
 	 *
 	 * @param patientId patient id to get the cohorts containing the patient
-	 * @param voided voided true/false whether or not to include voided patients
-	 * @return List&lt;Cohort&gt; object of matching Cohorts
+	 * @param includeVoidedCohorts voided true/false whether or not to include voided cohorts
+	 * @param includeVoidedMemberships voided true/false whether or not to include voided memberships
+	 *@param asOfDate  @return List&lt;Cohort&gt; object of matching Cohorts
 	 * @throws DAOException
 	 */
-	public List<Cohort> getCohortsContainingPatientId(Integer patientId, Boolean voided) throws DAOException;
-
+	public List<Cohort> getCohortsContainingPatientId(Integer patientId, Boolean includeVoidedCohorts,
```

Comment:
Remove and rename arguments based on the comments I added to the service method above